python/hail/expr/functions.py

```diff
@@ -340,16 +340,16 @@ def bind(f: Callable, *exprs):
 
 
 @typecheck(c1=expr_int32, c2=expr_int32, c3=expr_int32, c4=expr_int32)
-def chisq(c1, c2, c3, c4) -> StructExpression:
+def chi_sq_test(c1, c2, c3, c4) -> StructExpression:
```

Comment:
should this be chi_squared_test?

Reply:
We are indeed abbreviating `squared` with `sq`. R does `chisq.test`, `rchisq`, etc. In `scipy.stats` the test is `chisquare` whereas the distribution is `chi2`.

I could be convinced to expand it to `chi_squared_test` but I'm not inclined to expand `chi_sq` to `chi_squared` everywhere it's used as a statistic, so I found this version readable and consistent.

Reply:
ok, now I'm starting to like `chi_squared_test`, but still want to keep `chi_sq` for the reported statistic. Cool?

Reply:
fine with me